src/App.jsx

```diff
@@ -14,57 +15,79 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+const Good = memo(
+  ({ name = '', changeGood = () => {}, isActive = false }) => {
+    const goodClass = isActive ? 'has-background-success-light' : '';
+    const buttonClass = isActive ? 'button is-info' : 'button';
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+    return (
+      <tr data-cy="Good" className={goodClass}>
+        <td>
+          <button
+            data-cy={isActive ? 'RemoveButton' : 'AddButton'}
+            type="button"
+            className={buttonClass}
+            onClick={() => changeGood(name)}
+          >
+            {isActive ? '-' : '+'}
+          </button>
+        </td>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+        <td data-cy="GoodTitle" className="is-vcentered">
+          {name}
+        </td>
+      </tr>
+    );
+  },
+  (prevProps, nextProps) => prevProps.isActive === nextProps.isActive,
+);
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+const GoodsBoody = ({
```

Comment:
There is a typo in the component name `GoodsBoody`. It should likely be `GoodsBody` to reflect its purpose more accurately.